ShareX.IndexerLib/Forms/DirectoryIndexerForm.cs

```diff
@@ -98,18 +98,12 @@ private async Task IndexFolder()
                     {
                         tcMain.SelectedTab = tpPreview;
 
-                        if (Settings.Output == IndexerOutput.Html)
-                        {
-                            txtPreview.Visible = false;
-                            wbPreview.Visible = true;
-                            wbPreview.DocumentText = Source;
-                        }
-                        else
-                        {
-                            wbPreview.Visible = false;
-                            txtPreview.Visible = true;
-                            txtPreview.Text = Source;
-                        }
+                        bool isReadyForWebPreview = Settings.Output == IndexerOutput.Html;
+
+                        txtPreview.Visible = !isReadyForWebPreview;
+                        wbPreview.Visible = isReadyForWebPreview;
+
+                        (isReadyForWebPreview ? wbPreview.DocumentText : txtPreview.Text) = Source;
```

Comment:
doesn’t seem to be a valid syntax

Reply:
Hello @ficcialfaint , you're right, thanks for letting me know, that was my mistake to use JavaScript syntax in a C# code, so I have reverted changes in this commit.

Reply:
Got it, just saw that the tests are failing and then saw this